contributing_to_docs/doc_guidelines.adoc

```diff
@@ -1123,58 +1147,19 @@ $ oc adm create-error-template > errors.html
 ----
 ....
 
-* If your command contains multiple lines and uses callout annotations, you must comment out the callout(s) in the codeblock, as shown in the following example:
-+
-....
-To scale based on the percent of CPU utilization, create a `HorizontalPodAutoscaler` object for an existing object:
+// Similar guidance as with 1006
```

Comment:
Is this comment still required?

Reply:
No, it's not. I retained the comments just as reference until this PR is finalized. Planned to remove the comments before getting it merged.